apis/Google.Cloud.Firestore/Google.Cloud.Firestore/WatchStream.cs

```diff
@@ -200,7 +200,7 @@ async Task CloseStreamAsync()
                         // being dropped, at which point completing the stream will fail; we don't want the listener to stop at that
                         // point. Instead, it will reconnect.
                     }
-                    underlyingStream.GrpcCall.Dispose();
```

Comment:
nit: finally block here?

Reply:
Effectively done, but with a bit of refactoring. It's in a new commit.